CrazyCanvas/Include/GUI/LobbyGUI.h

```diff
@@ -43,32 +46,39 @@ class LobbyGUI : public Noesis::Grid
 	void OnButtonErrorOKClick(Noesis::BaseComponent* pSender, const Noesis::RoutedEventArgs& args);
 	void OnButtonHostGameClick(Noesis::BaseComponent* pSender, const Noesis::RoutedEventArgs& args);
 	
-	void StartSelectedServer(Noesis::Grid* pGrid);
+	void JoinSelectedServer(Noesis::Grid* pGrid);
 
 
 	bool OnLANServerFound(const LambdaEngine::ServerDiscoveredEvent& event);
+	bool OnClientConnected(const LambdaEngine::ClientConnectedEvent& event);
 
 	void FixedTick(LambdaEngine::Timestamp delta);
 
 private:
 	void SetRenderStagesActive();
 
-	void ErrorPopUp(ErrorCode errorCode);
+	void ErrorPopUp(PopUpCode errorCode);
+	void NotiPopUP(PopUpCode notificationCode);
+
 	void ErrorPopUpClose();
+	void NotiPopUpClose();
 
 	bool CheckServerStatus();
 	bool CheckServerSettings(const HostGameDescription& serverSettings);
 
-	void HostServer();
+	bool StartUpServer(std::string pApplicationName, std::string pCommandLine);
```

Comment:
Pass string by ref

Reply:
I probably forgot to change when using const char*